static/app/components/searchSyntax/grammar.pegjs

```diff
@@ -345,18 +346,24 @@ rel_date_format
 
 duration_format
   = value:numeric
-    unit:("ms"/"s"/"min"/"m"/"hr"/"h"/"day"/"d"/"wk"/"w")
+    unit:(duration_unit)
     &end_value {
       return tc.tokenValueDuration(value, unit);
     }
 
+duration_unit = "ms" / "s" / "min" / "m" / "hr" / "h" / "day" / "d" / "wk" / "w"
+
 size_format
   = value:numeric
-    unit:("bit"/"nb"/"bytes"/"kb"/"mb"/"gb"/"tb"/"pb"/"eb"/"zb"/"yb"/"kib"/"mib"/"gib"/"tib"/"pib"/"eib"/"zib"/"yib")
+    unit:(size_unit)
     &end_value {
       return tc.tokenValueSize(value, unit);
     }
 
+size_unit = bits / bytes
+bits = "bit"i / "kib"i / "mib"i / "gib"i / "tib"i / "pib"i / "eib"i / "zib"i / "yib"i
+bytes = "bytes"i / "nb"i / "kb"i / "mb"i / "gb"i / "tb"i / "pb"i / "eb"i / "zb"i / "yb"i
```

Comment:
👍 for these changes, would actually not mind seeing these pulled out into a separate PR

Reply:
Yup, lets do that